src/test/transaction_tests.cpp

```diff
@@ -38,6 +38,25 @@ extern UniValue read_json(const std::string &jsondata);
 
 BOOST_FIXTURE_TEST_SUITE(transaction_tests, BasicTestingSetup)
 
+BOOST_AUTO_TEST_CASE(tx_standard_creation)
```

Comment:
It's not clear to me what the purpose of this test is.

Is it a basic sanity check to see that we create a valid script? It also doesn't create transaction, so maybe it's in the wrong test suite?

If its purpose is to test minimaldata, then it looks like it covers these cases:
```
if ((b.size() == 1) && (b[0] >= 1 && b[0] <= 16))
```

```
else if ((b.size() == 1) && (b[0] == 0x81))
```

```
else if (b.size() < OP_PUSHDATA1)
```

Just to mention it, there is in `interpreter.cpp` a function `CheckMinimalPush` which can be used to test minimal data encoding without doing script evaluation.

Reply:
as you suspected the purpose is to test minimaldata encoding of vectors in the cases you indicated.  I've moved it to script_tests since you are right that's a better place for it.  CheckMinimalPush is currently a static function, and I don't think its a big deal to just run EvalScript so let's leave it like that.